models/list/list.go

```diff
@@ -38,7 +39,7 @@ func NewList(id int64, projectID int64, userID int64, title string, color string
 	return list
 }
 
-func FindList(projectID int64, listID int64) *ListStruct {
+func FindList(projectID int64, listID int64) (*ListStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindList should have comment or be unexported